fake/fake-extension.js

```diff
@@ -170,6 +170,13 @@ var seedMvcActions = (function() {
                         action: 'Index',
                         route: generate.common.route('home', 'index', null),
                         preActivities: [
+                            { access: 'middleware-start', correlationId: '123456789', name: 'middleware1', startTime: '2016-02-29T14:54:39.073 -0800' },
+                            { access: 'middleware-end', correlationId: '123456789', name: 'middleware1', endTime: '2016-02-29T14:54:40.073 -0800', duration: 1000, result: 'next' },
+                            { access: 'middleware-start', correlationId: '234567891', name: 'middleware2', startTime: '2016-02-29T14:54:40.073 -0800' },
+                            { access: 'mongo', type: 'data-mongodb-read', operation: 'toArray', query: {}, startTime: '2016-02-29T14:54:40.073 -0800', options: { find: 'MusicStore.AlbumCollection', limit: 0, skip: 0, query: {}, slaveOk: true, readPreference: { mode: 'primary' } }, connectionHost: 'localhost', connectionPort: 27017, database: 'MusicStore', collection: 'AlbumCollection' },
+                            { access: 'middleware-end', correlationId: '234567891', name: 'middleware2', endTime: '2016-02-29T14:54:41.073 -0800', duration: 1000, result: 'next' },
+                            { access: 'middleware-start', correlationId: '345678912', name: 'middleware3', startTime: '2016-02-29T14:54:41.073 -0800'},
+                            { access: 'middleware-end', correlationId: '345678912', name: 'middleware3', endTime: '2016-02-29T14:54:42.073 -0800', duration: 1000, result: 'end' },
```

Comment:
Typically I try and give things reasonably real titles... it saves from everything being View1-5 and Action1-15. Any chance we can get a little more creative with the names?